konfera/models/discountcodes.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from .tickettype import TicketType
+
+
+class DiscountCodes(models.Model):
+    title = models.CharField(max_length=128)
+    hash = models.CharField(max_length=64)
+    discount = models.IntegerField(max(100), min(0))
```

Comment:
I don't think that `max(100), min(0)` will work. Please use [validators](https://docs.djangoproject.com/en/1.10/ref/validators/#minvaluevalidator) instead. :)